Style header and main sections for the first part of the Portfolio Project

xandro2021 opened the pull request:
# Style the other home page sections in the Portfolio Project
in this PR:

+ Create the html structure for the projects' cards
+ Add the images and the rest of the styles regarding the works' cards
+ Update the current feacture branch with the branch from the last projects
+ Add the html structure and styles for the section about me
+ Create a fuctional drop down menu with HTML and CSS
+ Create the html for the contact section and style it

Thank you for reviewing my PR

xandro2021 committed:
Add initial html structure for the card that contains works

xandro2021 committed:
Add an auxiliary container with an auxiliary class

+ It was needed to apply a white background in the header. Otherwise
because of the header being 80% of the body, the other 20% would have
the default gray color from the template

xandro2021 committed:
Add missing general styles

+ It was missing the gray background color in the body
+ It was necessary to create an auxiliary class to apply white
background color to apply it in the header and the instroductory
section.
+ It was added a bottom left border radius to the instroductory section

xandro2021 committed:
Add image for the first card

xandro2021 committed:
Update the card styling and update general styles

+ the general bg color was updated
+ the default size of the container class was updated
+ it was applied the initial basic styles for the card containers and
the first card
+ It was updated the link refering to the styles.css in the index.html
+ It was added the general class container to the cards' container

xandro2021 committed:
Merge branch 'main' into finish-mobile-version

xandro2021 committed:
Adding missing html tags lost during the merge

xandro2021 committed:
Upload images used in the project's cards

xandro2021 committed:
Style the project's cards

xandro2021 committed:
Create the rest of the project's cards

xandro2021 committed:
Add HTML structure and styles for the section About Me

xandro2021 committed:
Create structure html for languages sections and get mobil image for the background

xandro2021 committed:
Style the about me and categories section

+ The about me was styled
+ The categories were given the styles accoring with the figma design
+ The drop menu from languages was completed

xandro2021 committed:
style the contact section and form

+ It was added a background with the design images
+ It was given similar styles than those use in the figma's template

Owusu-Desmond commented on index.html:
```diff
@@ -90,6 +92,227 @@ <h2>Let's connect</h2>
+                                <img src="../images/css.png" alt="javascript icon" />
+                                <p>CSS</p>
+                            </li>
+                        </ul>
```
- [ ] Please make sure that you make the languages here show on your page by default as it is in the Figma design,
### Your Design
![image](https://user-images.githubusercontent.com/92303891/219906234-befdb7f5-680a-407d-907f-7f8a31b1ed2f.png)
### Figma Design
![image](https://user-images.githubusercontent.com/92303891/219906260-881c52a0-d634-42e5-9376-ba2fbbdbe26a.png)

Owusu-Desmond commented on index.html:
```diff
@@ -90,6 +92,227 @@ <h2>Let's connect</h2>
+                        <ul class="languages extra-menu">
+                            <li>
+                                <img src="../images/javascript.png" alt="javascript icon" />
+                                <p>JavaScript</p>
```
- [x] Please for best practice's sake, Why don't you consider using `span` instead of using `p` element since this is not a paragraph? :smile:

xandro2021 replied:
yeap, I forgot the existence of span, I was deciding between a title, a div, or p because my mind was more busy thinking about how to make the drop-menu functional without JavaScript.
I'm fixing it.

Owusu-Desmond commented on index.html:
```diff
@@ -90,6 +92,227 @@ <h2>Let's connect</h2>
+                            </li>
+                            <li>
+                                <img src="../images/html.png" alt="javascript icon" />
+                                <p>HTML</p>
```
- [x] Please for best practice's sake, Why don't you consider using `span` instead of using the `p` element since this is not a paragraph? :smile:

Owusu-Desmond commented on index.html:
```diff
@@ -90,6 +92,227 @@ <h2>Let's connect</h2>
+                            </li>
+                            <li>
+                                <img src="../images/css.png" alt="javascript icon" />
+                                <p>CSS</p>
```
- [x] Please for best practice's sake, Why don't you consider using `span` instead of using the `p` element since this is not a paragraph? :smile:

Owusu-Desmond commented on index.html:
```diff
@@ -90,6 +92,227 @@ <h2>Let's connect</h2>
+
+            <section class="container">
+                <div class="contact-info">
+                    <h1>Contact Me</h1>
```
- [x] Since you have already used `h1` for `Hi, I'm Dagmawi` please consider using other heading tags like `h2`, `h3`, `h4`  .... :pray:

Owusu-Desmond reviewed (changes requested):
Hi @xandro2021 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<img src="https://media1.giphy.com/media/L422JoNlBBBD42zkQ9/giphy.gif"/>

## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

xandro2021 committed:
Follow first code reviewer recommendations

+ It was change all the <p> tags for <span> in the cases were it
contains only one Word
+ It was change the h1 in the contact info section for an h2

xandro2021 committed:
Disable hover effect (Languages menu always active)

franclobo commented on index.html:
```diff
@@ -90,6 +92,227 @@ <h2>Let's connect</h2>
+
+                </section>
+
+                <ul class="drop-buttons">
+                    <li class="drop-button">
+                        <span class="title">Languages</span>
+                        <ul class="languages extra-menu">
+                            <li>
+                                <img src="../images/javascript.png" alt="javascript icon" />
+                                <span>JavaScript</span>
+                            </li>
+                            <li>
+                                <img src="../images/html.png" alt="javascript icon" />
+                                <span>HTML</span>
+                            </li>
+                            <li>
+                                <img src="../images/css.png" alt="javascript icon" />
+                                <span>CSS</span>
+                            </li>
+                        </ul>
+                    </li>
+
+                    <li class="drop-button">
+                        <span class="title">Frameworks</span>
+                    </li>
+
+                    <li class="drop-button">
+                        <span class="title">Skills</span>
+
+                    </li>
+                </ul>
```
- [ ] Well done on doing the dropdown button, I would have approved it but the icons don't open as expected. please 🙏🏽  review the path of the image to correct the problem and look similar to the Figma template. Also, kindly change the `alt` attribute to have a different description according to the image.

![image](https://user-images.githubusercontent.com/58642949/219907617-0015ad8f-e32d-4c59-bb2b-e12020e50aad.png)

xandro2021 replied:
![workingFine](https://user-images.githubusercontent.com/88587370/219907983-08c5615a-1633-444d-9e2f-a8df758820e3.png)

I don´t get it. Mine is working fine without changes

xandro2021 replied:
![workingfine2](https://user-images.githubusercontent.com/88587370/219908045-4da3f8e1-b413-42b6-8d33-6a4d4faa9ff7.png)
@franclobo

xandro2021 replied:
okay, I see my mistake now. I was accessing it from the css folder because other icon somewhere were accessed from the CSS instead of the index

franclobo reviewed (changes requested):
Hi @xandro2021, 

# ♻️ Required Changes ♻️

![so-close-jim-halpert](https://user-images.githubusercontent.com/58642949/219907810-26cab2e2-6b14-4dac-b567-79c12a35c5cc.gif)

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- No linter errors ✔️ 
- Descriptive PR ✔️ 
- Do more than expected 💯 


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

xandro2021 committed:
Fix img address

emmiiorji reviewed (approved):
Hi @xandro2021, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


<div align='center'>
<img src='https://user-images.githubusercontent.com/20554583/219607923-71e0205b-a586-4d94-a603-aafadfbe59a4.gif'/>
</div>


## Good Highlights✔️

- Requested changes have been implemented

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
